content/posts/aws-autoscaling-groups-with-terraform.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Updating AWS autoscaling groups with Terraform and preserving desired capacity"
+date: 2020-06-21
+authors:
+  - author:
+      name: "Ariel Peltz"
+      link: "https://github.com/arielpeltz"
+tags: ["aws", "terraform", "autoscaling", "desired_capacity"]
+---
+
```

Comment:
There is a lot of switching between past and present tense that makes it difficult to read.

Reply:
You mean generally in the document?

Reply:
Hopefully fixed